{"uid":"c08c8a5890eedc56","name":"Страница индивидуальной контрольной: результат тестирования из попытках зависеть от настройки 'За лучшую попытку'","fullName":"OdinUiTests.Tests.Activity.TestActivityStatusWithTwoOrMoreAttemptsTests.Страница индивидуальной контрольной: результат тестирования из попытках зависеть от настройки 'За лучшую попытку'","historyId":"OdinUiTests.Tests.Activity.TestActivityStatusWithTwoOrMoreAttemptsTests.Страница индивидуальной контрольной: результат тестирования из попытках зависеть от настройки 'За лучшую попытку'","time":{"start":1691036091119,"stop":1691036091504,"duration":385},"status":"failed","statusMessage":"System.AggregateException : One or more errors occurred. (The SSL connection could not be established, see inner exception.)\n  ----> System.Net.Http.HttpRequestException : The SSL connection could not be established, see inner exception.\n  ----> System.Security.Authentication.AuthenticationException : The remote certificate is invalid according to the validation procedure: RemoteCertificateNameMismatch, RemoteCertificateChainErrors","statusTrace":"   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)\n   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)\n   at OdinUiTests.Services.AccountService.GetAccessToken(UsersEnum user) in /azp/_work/1/s/Services/AccountService.cs:line 20\n   at OdinUiTests.Services.ActivityService.GetActivityTestInfo(String actId) in /azp/_work/1/s/Services/ActivityService.cs:line 117\n   at OdinUiTests.Services.ExamineeService.GetActivityExamineesId(String actId) in /azp/_work/1/s/Services/ExamineeService.cs:line 17\n   at OdinUiTests.Services.ExamineeService.CheckTestHasNoAttempts(String actId) in /azp/_work/1/s/Services/ExamineeService.cs:line 73\n   at OdinUiTests.Tests.Activity.TestActivityStatusWithTwoOrMoreAttemptsTests.ResultOfPassingDependOnSettings(String actId, Int32 evaluation, String attemptSetting) in /azp/_work/1/s/Tests/Activity/TestActivityTests.cs:line 669\n--HttpRequestException\n   at RestSharp.RestClientExtensions.PostAsync(IRestClient client, RestRequest request, CancellationToken cancellationToken)\n--AuthenticationException\n   at System.Net.Security.SslStream.SendAuthResetSignal(ProtocolToken message, ExceptionDispatchInfo exception)\n   at System.Net.Security.SslStream.CompleteHandshake(SslAuthenticationOptions sslAuthenticationOptions)\n   at System.Net.Security.SslStream.ForceAuthenticationAsync[TIOAdapter](TIOAdapter adapter, Boolean receiveFirst, Byte[] reAuthenticationData, Boolean isApm)\n   at System.Net.Http.ConnectHelper.EstablishSslConnectionAsync(SslClientAuthenticationOptions sslOptions, HttpRequestMessage request, Boolean async, Stream stream, CancellationToken cancellationToken)","flaky":false,"newFailed":true,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"OneTimeSetUp","time":{"start":1691036091504,"stop":1691036091504,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"SetUp","time":{"start":1691036091504,"stop":1691036091504,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"testStage":{"status":"failed","statusMessage":"System.AggregateException : One or more errors occurred. (The SSL connection could not be established, see inner exception.)\n  ----> System.Net.Http.HttpRequestException : The SSL connection could not be established, see inner exception.\n  ----> System.Security.Authentication.AuthenticationException : The remote certificate is invalid according to the validation procedure: RemoteCertificateNameMismatch, RemoteCertificateChainErrors","statusTrace":"   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)\n   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)\n   at OdinUiTests.Services.AccountService.GetAccessToken(UsersEnum user) in /azp/_work/1/s/Services/AccountService.cs:line 20\n   at OdinUiTests.Services.ActivityService.GetActivityTestInfo(String actId) in /azp/_work/1/s/Services/ActivityService.cs:line 117\n   at OdinUiTests.Services.ExamineeService.GetActivityExamineesId(String actId) in /azp/_work/1/s/Services/ExamineeService.cs:line 17\n   at OdinUiTests.Services.ExamineeService.CheckTestHasNoAttempts(String actId) in /azp/_work/1/s/Services/ExamineeService.cs:line 73\n   at OdinUiTests.Tests.Activity.TestActivityStatusWithTwoOrMoreAttemptsTests.ResultOfPassingDependOnSettings(String actId, Int32 evaluation, String attemptSetting) in /azp/_work/1/s/Tests/Activity/TestActivityTests.cs:line 669\n--HttpRequestException\n   at RestSharp.RestClientExtensions.PostAsync(IRestClient client, RestRequest request, CancellationToken cancellationToken)\n--AuthenticationException\n   at System.Net.Security.SslStream.SendAuthResetSignal(ProtocolToken message, ExceptionDispatchInfo exception)\n   at System.Net.Security.SslStream.CompleteHandshake(SslAuthenticationOptions sslAuthenticationOptions)\n   at System.Net.Security.SslStream.ForceAuthenticationAsync[TIOAdapter](TIOAdapter adapter, Boolean receiveFirst, Byte[] reAuthenticationData, Boolean isApm)\n   at System.Net.Http.ConnectHelper.EstablishSslConnectionAsync(SslClientAuthenticationOptions sslOptions, HttpRequestMessage request, Boolean async, Stream stream, CancellationToken cancellationToken)","steps":[{"name":"Страница индивидуальной контрольной: результат тестирования из попытках зависеть от настройки 'За лучшую попытку'","time":{"start":1691036091119,"stop":1691036091504,"duration":385},"status":"failed","statusMessage":"System.AggregateException : One or more errors occurred. (The SSL connection could not be established, see inner exception.)\n  ----> System.Net.Http.HttpRequestException : The SSL connection could not be established, see inner exception.\n  ----> System.Security.Authentication.AuthenticationException : The remote certificate is invalid according to the validation procedure: RemoteCertificateNameMismatch, RemoteCertificateChainErrors","statusTrace":"   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)\n   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)\n   at OdinUiTests.Services.AccountService.GetAccessToken(UsersEnum user) in /azp/_work/1/s/Services/AccountService.cs:line 20\n   at OdinUiTests.Services.ActivityService.GetActivityTestInfo(String actId) in /azp/_work/1/s/Services/ActivityService.cs:line 117\n   at OdinUiTests.Services.ExamineeService.GetActivityExamineesId(String actId) in /azp/_work/1/s/Services/ExamineeService.cs:line 17\n   at OdinUiTests.Services.ExamineeService.CheckTestHasNoAttempts(String actId) in /azp/_work/1/s/Services/ExamineeService.cs:line 73\n   at OdinUiTests.Tests.Activity.TestActivityStatusWithTwoOrMoreAttemptsTests.ResultOfPassingDependOnSettings(String actId, Int32 evaluation, String attemptSetting) in /azp/_work/1/s/Tests/Activity/TestActivityTests.cs:line 669\n--HttpRequestException\n   at RestSharp.RestClientExtensions.PostAsync(IRestClient client, RestRequest request, CancellationToken cancellationToken)\n--AuthenticationException\n   at System.Net.Security.SslStream.SendAuthResetSignal(ProtocolToken message, ExceptionDispatchInfo exception)\n   at System.Net.Security.SslStream.CompleteHandshake(SslAuthenticationOptions sslAuthenticationOptions)\n   at System.Net.Security.SslStream.ForceAuthenticationAsync[TIOAdapter](TIOAdapter adapter, Boolean receiveFirst, Byte[] reAuthenticationData, Boolean isApm)\n   at System.Net.Http.ConnectHelper.EstablishSslConnectionAsync(SslClientAuthenticationOptions sslOptions, HttpRequestMessage request, Boolean async, Stream stream, CancellationToken cancellationToken)","steps":[],"attachments":[{"uid":"63603150847baf19","name":"ResultOfPassingDependOnSettings.jpg","source":"63603150847baf19.jpg","type":"image/jpeg","size":303229},{"uid":"705b5afac4bcc2e1","name":"Console Output","source":"705b5afac4bcc2e1.txt","type":"text/plain","size":30}],"parameters":[],"stepsCount":0,"attachmentsCount":2,"shouldDisplayMessage":true,"hasContent":true}],"attachments":[],"parameters":[],"stepsCount":1,"attachmentsCount":2,"shouldDisplayMessage":false,"hasContent":true},"afterStages":[{"name":"TearDown","time":{"start":1691036091504,"stop":1691036091504,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"labels":[{"name":"thread","value":"ParallelWorker#5"},{"name":"host","value":"7913e481e174"},{"name":"package","value":"OdinUiTests.Tests.Activity"},{"name":"testMethod","value":"ResultOfPassingDependOnSettings"},{"name":"testClass","value":"TestActivityStatusWithTwoOrMoreAttemptsTests"},{"name":"tag","value":"Активность"},{"name":"tag","value":"Регресс"},{"name":"suite","value":"Активность"},{"name":"subSuite","value":"Контрольная активность"},{"name":"severity","value":"critical"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"Param #0","value":"5209"},{"name":"Param #1","value":"1"},{"name":"Param #2","value":"bestTry"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"critical","retries":[],"categories":[{"name":"Test Defects","messageRegex":".*","matchedStatuses":["failed"],"flaky":false}],"history":{"statistic":{"failed":3,"broken":0,"skipped":0,"passed":15,"unknown":0,"total":18},"items":[{"uid":"3b6636e46b19dd56","status":"passed","statusDetails":"Страница индивидуальной контрольной: результат тестирования из попытках зависеть от настройки 'За лучшую попытку'","time":{"start":1690951449721,"stop":1690951474287,"duration":24566}},{"uid":"bf8fdd8c6ae240b5","status":"passed","statusDetails":"Страница индивидуальной контрольной: результат тестирования из попытках зависеть от настройки 'За лучшую попытку'","time":{"start":1690864734180,"stop":1690864754444,"duration":20264}},{"uid":"2aac51d3472a3cb7","status":"passed","statusDetails":"Страница индивидуальной контрольной: результат тестирования из попытках зависеть от настройки 'За лучшую попытку'","time":{"start":1690778253809,"stop":1690778269418,"duration":15609}},{"uid":"a9538a7a053bef3c","status":"passed","statusDetails":"Страница индивидуальной контрольной: результат тестирования из попытках зависеть от настройки 'За лучшую попытку'","time":{"start":1690518738197,"stop":1690518752802,"duration":14605}},{"uid":"d8f7482ba433bf9f","status":"passed","statusDetails":"Страница индивидуальной контрольной: результат тестирования из попытках зависеть от настройки 'За лучшую попытку'","time":{"start":1690432729134,"stop":1690432744312,"duration":15178}},{"uid":"7ce292d1e57a8553","status":"passed","statusDetails":"Страница индивидуальной контрольной: результат тестирования из попытках зависеть от настройки 'За лучшую попытку'","time":{"start":1690173462284,"stop":1690173482039,"duration":19755}},{"uid":"be1a95f04168f759","status":"failed","statusDetails":"System.AggregateException : One or more errors occurred. (Request timed out)\n  ----> System.TimeoutException : Request timed out\n  ----> System.Threading.Tasks.TaskCanceledException : The request was canceled due to the configured HttpClient.Timeout of 100 seconds elapsing.\n  ----> System.TimeoutException : A task was canceled.\n  ----> System.Threading.Tasks.TaskCanceledException : A task was canceled.","time":{"start":1689914188507,"stop":1689914288890,"duration":100383}},{"uid":"4cb8d140b7d5ab20","status":"passed","statusDetails":"Страница индивидуальной контрольной: результат тестирования из попытках зависеть от настройки 'За лучшую попытку'","time":{"start":1689827332611,"stop":1689827357538,"duration":24927}},{"uid":"67c736464943eb5c","status":"passed","statusDetails":"Страница индивидуальной контрольной: результат тестирования из попытках зависеть от настройки 'За лучшую попытку'","time":{"start":1689741573477,"stop":1689741589149,"duration":15672}},{"uid":"db20180cdf95eab9","status":"passed","statusDetails":"Страница индивидуальной контрольной: результат тестирования из попытках зависеть от настройки 'За лучшую попытку'","time":{"start":1689656545072,"stop":1689656561094,"duration":16022}},{"uid":"489707897612f151","status":"passed","statusDetails":"Страница индивидуальной контрольной: результат тестирования из попытках зависеть от настройки 'За лучшую попытку'","time":{"start":1689570178685,"stop":1689570194503,"duration":15818}},{"uid":"bc658609faa09901","status":"passed","statusDetails":"Страница индивидуальной контрольной: результат тестирования из попытках зависеть от настройки 'За лучшую попытку'","time":{"start":1689310324645,"stop":1689310341559,"duration":16914}},{"uid":"c7a5aa0a25b85d","status":"passed","statusDetails":"Страница индивидуальной контрольной: результат тестирования из попытках зависеть от настройки 'За лучшую попытку'","time":{"start":1689223404500,"stop":1689223418952,"duration":14452}},{"uid":"25b82e3ded5c898d","status":"passed","statusDetails":"Страница индивидуальной контрольной: результат тестирования из попытках зависеть от настройки 'За лучшую попытку'","time":{"start":1689140933599,"stop":1689140953181,"duration":19582}},{"uid":"cc751c5ace6cd0a","status":"failed","statusDetails":"System.AggregateException : One or more errors occurred. (Request failed with status code GatewayTimeout)\n  ----> System.Net.Http.HttpRequestException : Request failed with status code GatewayTimeout","time":{"start":1689139193783,"stop":1689139209057,"duration":15274}},{"uid":"f1f3163edfa339dc","status":"passed","statusDetails":"Страница индивидуальной контрольной: результат тестирования из попытках зависеть от настройки 'За лучшую попытку'","time":{"start":1689050059044,"stop":1689050077993,"duration":18949}},{"uid":"e1e2975153ab6d43","status":"passed","statusDetails":"Страница индивидуальной контрольной: результат тестирования из попытках зависеть от настройки 'За лучшую попытку'","time":{"start":1688964646149,"stop":1688964669708,"duration":23559}}]},"tags":["Регресс","Активность"]},"source":"c08c8a5890eedc56.json","parameterValues":["5209","1","bestTry"]}